examples/pytorch/rgcn/dataset.py

```diff
@@ -0,0 +1,535 @@
+""" Knowledge graph dataset for Relational-GCN
+Code adapted from authors' implementation of Relational-GCN
+https://github.com/tkipf/relational-gcn
+https://github.com/MichSchli/RelationPrediction
+"""
+
+from __future__ import print_function
+from __future__ import absolute_import
+import numpy as np
+import scipy.sparse as sp
+import os, gzip
+import rdflib as rdf
+import pandas as pd
+from collections import Counter
+import re
+
+from dgl.data.utils import download, extract_archive, get_download_dir, _get_dgl_url
+
+np.random.seed(123)
+_downlaod_prefix = _get_dgl_url('dataset/')
+
+class RGCNEntityDataset(object):
```

Comment:
why do we need another data loader? i thought we loaded these graphs in both DGLGraph and DGLHeteroGraph.

Reply:
This allows loading data in DGLHeteroGraph and use to_homo to generate homogeneous graph.